### PR TITLE
Fixup keychron/q3

### PR DESCRIPTION
**Pull request by waffle87**
## Description

As title — to align with develop specific changes.
Also changes `rgb_matrix_indicators_advanced_user` -> `rgb_matrix_indicators_advanced_kb`
Unsure what `KC_INT1PT(N)` was, so changed to `KC_INT1`

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by tzarc** [comment]
There are a few cases where `KC_ROPT` and `KC_ROPTN` have potentially been changed incorrectly, probably should rectify that.

File: keyboards/keychron/q3/jis_encoder/keymaps/default/keymap.c [hunk 1]
```diff
@@ -26,11 +26,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_INT1,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis_encoder/keymaps/default/keymap.c:33]:
```suggestion
        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

File: keyboards/keychron/q3/jis_encoder/keymaps/keychron/keymap.c [hunk 2]
```diff
@@ -27,11 +27,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_INT1, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis_encoder/keymaps/keychron/keymap.c:34]:
```suggestion
        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

File: keyboards/keychron/q3/jis_encoder/keymaps/via/keymap.c [hunk 3]
```diff
@@ -26,11 +26,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_INT1,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis_encoder/keymaps/via/keymap.c:33]:
```suggestion
        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

File: keyboards/keychron/q3/jis/keymaps/via/keymap.c [hunk 4]
```diff
@@ -26,11 +26,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_INT1,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis/keymaps/via/keymap.c:33]:
```suggestion
        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

File: keyboards/keychron/q3/jis/keymaps/keychron/keymap.c [hunk 5]
```diff
@@ -27,11 +27,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_INT1, MO(MAC_FN), KC_RCTL,             KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis/keymaps/keychron/keymap.c:34]:
```suggestion
        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

File: keyboards/keychron/q3/jis/keymaps/default/keymap.c [hunk 6]
```diff
@@ -26,11 +26,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_INT1,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis/keymaps/default/keymap.c:33]:
```suggestion
        KC_LCTL,  KC_LOPT,  KC_LCMD,  KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMD,  KC_ROPT,  MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by waffle87** [comment]
Thanks fauxpark

File: keyboards/keychron/q3/jis/keymaps/keychron/keymap.c [hunk 7]
```diff
@@ -27,11 +27,11 @@ enum layers{
...
         KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,                        KC_DEL,   KC_END,   KC_PGDN,
         KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,    KC_ENT,
-        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RO,      KC_RSFT,                      KC_UP,
-        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_HANJ,                      KC_SPC,                       KC_HAEN,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
+        KC_LSFT,            KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_INT1,      KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,             KC_LEFT,  KC_DOWN,  KC_RGHT),
```
Comment (fauxpark) [inline keyboards/keychron/q3/jis/keymaps/keychron/keymap.c:34]:
```suggestion
        KC_LCTL,  KC_LOPTN, KC_LCMMD, KC_LNG2,                      KC_SPC,                       KC_LNG1,  KC_RCMMD, KC_ROPTN, MO(MAC_FN), KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT),
```

**Review by fauxpark — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**